packages/tpetra/core/test/CrsMatrix/CrsMatrix_ReplaceDiagonal.cpp

```diff
@@ -155,21 +155,29 @@ namespace { // (anonymous)
          * the number of replaced diagonal entries to match the
          * local number of rows.
          */
-        TEST_EQUALITY(numReplacedDiagEntries, matrix->getNodeNumRows());
+	const LO lclNumRows = static_cast<LO> (matrix->getNodeNumRows ());
+        TEST_EQUALITY(numReplacedDiagEntries, lclNumRows);
```

Comment:
Does this indentation not quite line up here?  Looks like TEST_EQUALITY() is indented one space too many.